common/src/main/scala/hmda/parser/filing/lar/LarParser.scala

```diff
@@ -10,16 +10,21 @@ import scala.util.{Failure, Success, Try}
 
 trait LarParser {
 
-  def validateIntField(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[Int] =
+  def validateIntFieldReturnString(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[String] =
+    Try(value.toInt) match {
+      case Success(i) => value.validNel
+      case Failure(e) => parserValidationError.invalidNel
+    }
+
+  def validateIntFieldReturnInt(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[Int] =
     Try(value.toInt) match {
       case Success(i) => i.validNel
-      case Failure(e) =>
-        parserValidationError.invalidNel
+      case Failure(e) => parserValidationError.invalidNel
     }
 
-  def validateDoubleField(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[Double] =
+  def validateDoubleFieldReturnString(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[String] =
     Try(value.toDouble) match {
-      case Success(i) => i.validNel
+      case Success(i) => value.validNel
```

Comment:
if user actually put in scientific notation, do we want to prevent it? I'm wondering if we might want to go the regex route?